External/FEXCore/Source/Interface/Context/Context.h

```diff
@@ -155,11 +155,13 @@ namespace FEXCore::Context {
     void RegisterHostSignalHandler(int Signal, HostSignalDelegatorFunction Func, bool Required);
     void RegisterFrontendHostSignalHandler(int Signal, HostSignalDelegatorFunction Func, bool Required);
 
-    static void RemoveCodeEntry(FEXCore::Core::InternalThreadState *Thread, uint64_t GuestRIP);
+    // Must be called from owning thread
```

Comment:
Owning what though? The `Context` or the `CpuStateFrame`?

Reply:
Owning the `InternalThreadState` (CpuStateFrame->Thread)